apps/www/app/api-v2/ticket-og/route.tsx

```diff
@@ -0,0 +1,555 @@
+import React from 'react'
+import { ImageResponse } from '@vercel/og'
+import { createClient } from '@supabase/supabase-js'
+import { themes } from '~/components/LaunchWeek/12/Ticket/ticketThemes'
+
+export const runtime = 'edge' // 'nodejs' is the default
+export const dynamic = 'force-dynamic' // defaults to auto
+export const fetchCache = 'force-no-store'
+export const revalidate = 0
+
+const corsHeaders = {
+  'Access-Control-Allow-Origin': '*',
+  'Access-Control-Allow-Headers': 'authorization, x-client-info, apikey, content-type',
+}
+
+const SUPABASE_URL = process.env.NEXT_PUBLIC_SUPABASE_URL
+
+const STORAGE_URL = `${SUPABASE_URL}/storage/v1/object/public/images/launch-week/lw12`
+
+// Load custom font
+const FONT_URL = `${STORAGE_URL}/assets/font/CircularStd-Book.otf`
+const MONO_FONT_URL = `${STORAGE_URL}/assets/font/SourceCodePro-Regular.ttf`
+const font = fetch(new URL(FONT_URL, import.meta.url)).then((res) => res.arrayBuffer())
+const mono_font = fetch(new URL(MONO_FONT_URL, import.meta.url)).then((res) => res.arrayBuffer())
+// const BUCKET_FOLDER_VERSION = 'v1'
+
+const LW_TABLE = 'tickets'
+const LW_MATERIALIZED_VIEW = 'tickets_view'
+
+export async function GET(req: Request, res: Response) {
+  const url = new URL(req.url)
+  console.log(process.env.NEXT_PUBLIC_SUPABASE_URL)
```

Comment:
Are all these console logs in this file needed? Lets remove them otherwise

Reply:
oops. ill remove